lib/legalizer.rb

```diff
@@ -31,27 +28,36 @@ def self.split_year(user)
       return nil
     end
 
+    current_year_request_total = 0
+    next_year_request_total = 0 
 
-    current_year_balence = 0
-    next_year_balenence = 0
+    user.pto_requests.each do |r|
+      if r.request_date.year == current_year
+        current_year_request_total += r.cost
+      elsif r.request_date.year == current_year + 1
+        next_year_request_total += r.cost
+      end
+    end
+
+    current_year_balance = 0
+    next_year_balance = 0
 
     # seperate points
     case current_quarter
     when 1
-      current_year_balence = user.bank_value
+      current_year_balance = user.bank_value
     when 2
-      current_year_balence = user.bank_value - 45
-      next_year_balenence = 45 - next_year_total
+      current_year_balance = 45 - current_year_request_total
```

Comment:
if we're going this route, current_year_balence will always need to be determined by subtraction from 180 regardless of the quarter

Reply:
however +1 on explicitly calculating current year balance, rather than just inferring it